django/contrib/admin/utils.py

```diff
@@ -54,10 +54,17 @@ def lookup_spawns_duplicates(opts, lookup_path):
     return False
 
 
+def get_value_from_parameters(parameters, key):
+    value = parameters.get(key)
+    return value[-1] if isinstance(value, list) else value
+
+
 def prepare_lookup_value(key, value, separator=","):
     """
     Return a lookup value prepared to be used in queryset filtering.
     """
+    if isinstance(value, list):
+        value = value[-1]
```

Comment:
I think we should handle `__in` with list of values.

Reply:
This is the first commit, I think the `__in` case looks like `["1,2,3"]` and then it gets changed in the next bit